torchvision/models/segmentation/deeplabv3.py

```diff
@@ -220,7 +220,7 @@ def _deeplabv3_mobilenetv3(
     backbone = IntermediateLayerGetter(backbone, return_layers=return_layers)
 
     aux_classifier = FCNHead(aux_inplanes, num_classes) if aux else None
-    classifier = DeepLabHead(out_inplanes, num_classes)
+    classifier = DeepLabHead(out_inplanes, num_classes, atrous_rates=(6, 12, 18))
```

Comment:
Unfortunately this would be BC-breaking. We'll have to preserve the values that are currently used here (relying on the default).

```suggestion
    classifier = DeepLabHead(out_inplanes, num_classes)
```